### ISSUE TITLE
require is not defined

### ISSUE THREAD

**Issue by quiqueck** [pyodide/pyodide#1880]
Just updated my pyodide from 0.17.0 runnig from a WebWorker to 0.18.1. Since I updated I get the following error when calling `loadPyodide` from the WebWorker:

```
Uncaught (in promise) ReferenceError: require is not defined
    at fetchRemotePackage (pyodide.asm.js:14)
    at loadPackage (pyodide.asm.js:14)
    at pyodide.asm.js:14
    at pyodide.asm.js:14
    at loadPyodide (pyodide.js:227)
    at async listener (pyWorker.js:78)
```

Looks like `fetchRemotePackage`was changed in 0.18 to use `require('fs')` when `process` is defined as an Object (which it is in a WebWorker).

**Comment by rth** [comment]
Yes, indeed thanks for the report. I initially backported https://github.com/emscripten-core/emscripten/pull/11785 to add Node support, but we also should backport https://github.com/emscripten-core/emscripten/pull/14501 to resolve this. Or better -- update emscripten version.

**Comment by ryanking13** [comment]
We are not using the problematic plugin now, so I think the error will not happen anymore. Closing as resolved, but don't hesitate to reopen if you still have the issue.
